config.yaml

```diff
@@ -0,0 +1,11 @@
+options:
+  cuda-version:
+    type: string
+    default: "8.0.61"
+    description: |
+      The version of CUDA to be intalled.
+  cuda-subversion:
```

Comment:
Oh, I missed that one. But yes, let's split on the last '-'. I think that's better than having two config options.